io.github.lxi-tools.yaml

```diff
@@ -0,0 +1,113 @@
+---
+app-id: io.github.lxi-tools.lxi_gui
```

Comment:
You forgot to rename the manifest, it should match the app-id

Reply:
Fixed.